MAUI/TreeMap/levels.md

```diff
@@ -0,0 +1,689 @@
+/// </summary>
+public class PopulationViewModel
+{
+    public PopulationViewModel()
+    {
+        this.PopulationDetails = new ObservableCollection<PopulationDetails>()
+        {
+            new PopulationDetails() { Continent ="North America", Country = "United States of America", Population = 339996564 },
+            new PopulationDetails() { Continent ="South America", Country = "Brazil", Population = 216422446 },
+            new PopulationDetails() { Continent ="North America", Country = "Mexico", Population = 128455567 },
+            new PopulationDetails() { Continent ="South America", Country = "Colombia", Population = 52085168 },
+            new PopulationDetails() { Continent ="South America", Country = "Argentina", Population = 45773884 },
+            new PopulationDetails() { Continent ="North America", Country = "Canada", Population = 38781292 },
+            new PopulationDetails() { Continent ="South America", Country = "Peru", Population = 34352719 },
+            new PopulationDetails() { Continent ="South America", Country = "Venezuela", Population = 28838499 },
+            new PopulationDetails() { Continent ="South America", Country = "Chile", Population = 19629590 },
+            new PopulationDetails() { Continent ="South America", Country = "Ecuador", Population = 18190484 },
+            new PopulationDetails() { Continent ="North America", Country = "Guatemala", Population = 18092026 },
+            new PopulationDetails() { Continent ="South America", Country = "Bolivia", Population = 12388571 },
+            new PopulationDetails() { Continent ="North America", Country = "Honduras", Population = 10593798 },
+            new PopulationDetails() { Continent ="North America", Country = "Nicaragua", Population = 7046311 },
+            new PopulationDetails() { Continent ="South America", Country = "Paraguay", Population = 6861524 },
+            new PopulationDetails() { Continent ="North America", Country = "El Salvador", Population = 6364943 },
+            new PopulationDetails() { Continent ="North America", Country = "Costa Rica", Population = 5212173 },
+            new PopulationDetails() { Continent ="South America", Country = "Uruguay", Population = 3423109 },
+        };
+    }
+
+    public ObservableCollection<PopulationDetails> PopulationDetails
+    {
+        get;
+        set;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+N> Setting a background value for the property applies the same background color to tree map header items within the same level. To customize the background color for both header and leaf items at each level, utilize the `GroupItemBrushSettings` property.
+
+### Customize level appearance using stroke
+
+The `Stroke` property of `TreeMapLevel` can be customized by using the `Levels` in the `SfTreeMap.` It is used to customize the stroke color for the tree map header item. The default value of `Stroke` is `new SolidColorBrush(Color.FromArgb("#CAC4D0")).`
```

Comment:
The `Stroke` property of `TreeMapLevel` can be customized using the `Levels` in the `SfTreeMap.` It is used to customize the stroke color for the treemap header item. The default value of `Stroke` is `new SolidColorBrush(Color.FromArgb("#CAC4D0")).`